stable/wordpress/values-production.yaml

```diff
@@ -63,6 +63,17 @@ wordpressTablePrefix: wp_
 ##
 wordpressSkipInstall: false
 
+## Set up update strategy for wordpress installation. Set to Recreate if you use persistent volume that cannot be mounted by more than one pods to makesure the pods is destroyed first.
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+## Example:
+## wordpressUpdateStrategy:
+##  type: RollingUpdate
+##  rollingUpdate:
+##    maxSurge: 25%
+##    maxUnavailable: 25%
+wordpressUpdateStrategy:
```

Comment:
I'd use `updateStrategy` as name since we're using `wordpressXXX` for properties that affect directly WordPress (the application) configuration. However, this property affect the K8s deployment definition.

Reply:
Done